units/src/amount/unsigned.rs

```diff
@@ -405,6 +405,12 @@ impl Amount {
         SignedAmount::from_sat(self.to_sat() as i64) // Cast ok, signed amount and amount share positive range.
             .expect("range of Amount is within range of SignedAmount")
     }
+    /// Prevents overflow in the subtraction
+    #[must_use]
+    pub fn signed_sub(self, rhs: Self) -> SignedAmount {
+        (self.to_signed() - rhs.to_signed())
+        .expect("difference of two amounts is always within SignedAmount range")
+    }
```

Comment:
`units` is currently stabalising so we are being super careful with it. Can we have it like this please. (Note the new line separating the function from the previous one.)


```rust

    /// Infallibly subtracts one `Amount` from another returning a [`SignedAmount`].
    ///
    /// Since `SignedAmount::MIN` is equivalent to `-Amount::MAX` subtraction of two signed amounts
    /// can never overflow a `SignedAmount`.
    #[must_use]
    pub fn signed_sub(self, rhs: Amount) -> SignedAmount {
        (self.to_signed() - rhs.to_signed())
            .expect("difference of two amounts is always within SignedAmount range")
    }
```